app-admin/bagger-tools/bagger-tools-0.4.ebuild

```
@@ -0,0 +1,70 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit perl-module
+
+DESCRIPTION="adjust bagger tools"
+HOMEPAGE="https://github.com/adjust/bagger"
+SLOT="0"
+IUSE="schaufel bagger master"
+LICENSE="Unlicense"
+
+if [[ ${PV} == 9999 ]]
+then
+	inherit git-r3
+	KEYWORDS=""
+	EGIT_REPO_URI="https://github.com/adjust/bagger.git"
+else
+	inherit vcs-snapshot
+	SRC_URI="https://github.com/adjust/${PN}/archive/v${PVR}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
```

```suggestion
      KEYWORDS="amd64"
```
Our software releases don't need to be keyworded I think. ;)

---

I think we are keywording btw.  We might want to rethink our policies there though.  I will leave that up to you and David